apps/meteor/client/views/omnichannel/triggers/TriggersTableContainer.tsx

```diff
@@ -0,0 +1,33 @@
+import { Callout } from '@rocket.chat/fuselage';
+import { useEndpoint, useTranslation } from '@rocket.chat/ui-contexts';
+import { useQuery } from '@tanstack/react-query';
+import React, { useState } from 'react';
+
+import type { GenericTableParams } from '../../../components/GenericTable/GenericTable';
+import TriggersTable from './TriggersTable';
+
+const TriggersTableContainer = () => {
+	const t = useTranslation();
+	const [params, setParams] = useState<GenericTableParams>(() => ({ current: 0, itemsPerPage: 25 }));
+
+	const { current, itemsPerPage } = params;
+
+	const getTriggers = useEndpoint('GET', '/v1/livechat/triggers');
+	const { data, refetch, isError } = useQuery(['/v1/livechat/triggers'], () => getTriggers({ offset: current, count: itemsPerPage }));
```

Comment:
```suggestion
	const { data, refetch, isError } = useQuery(['/v1/livechat/triggers', current, itemsPerPage], () =>
		getTriggers({ offset: current, count: itemsPerPage }),
	);
```

I believe that you will need to pass the `current` and `itemPerPage` here as a key.